receiver/podmanreceiver/receiver_test.go

```diff
@@ -74,14 +68,14 @@ func TestScraperLoop(t *testing.T) {
 		}
 	}()
 
-	assert.NoError(t, r.Start(ctx, componenttest.NewNopHost()))
+	assert.NoError(t, r.start(ctx, componenttest.NewNopHost()))
+	defer assert.NoError(t, r.shutdown(ctx))
```

Comment:
```suggestion
	defer func(){ assert.NoError(t, r.shutdown(ctx)) }()
```
I realize the test is currently passing, but without the anonymous function `shutdown` is evaluated at this line, rather than being properly deferred.